### ISSUE TITLE
Creating a formal Green Software Foundation response to the GHG Protocol survey

### ISSUE THREAD

**Issue by mrchrisadams** [Green-Software-Foundation/policy-wg#34]
THe GHG protocol is currently being reviewed, specifically with respect to how scope 2 is used for measuring the environmental impact of activity:

> Greenhouse Gas Protocol is starting a process to collect stakeholder input to understand the need, scope, and potential approaches to inform updates or additional guidance related to GHG Protocol’s Corporate Standard, Scope 2 Guidance, Scope 3 Standard, and supporting documents.
> The aim of any updates will be to align with best practice approaches to ensure GHG Protocol standards for scope 1, scope 2 and scope 3 are effective in providing a rigorous and credible accounting foundation for businesses to measure, plan and track progress toward science-based and net-zero targets in line with the global 1.5°C goal. Any future updates will seek harmonization and alignment with accounting rules under development through major disclosure initiatives.

While the SCI does not currently try to be compatible with the GHG protocol's standard it's still the defacto standard in many parts of the world, and it seems a very good idea to collate a response from the GSF, or at least some guidance on how people should see the two approaches.

https://ghgprotocol.org/survey-need-ghg-protocol-corporate-standards-and-guidance-updates

There is a template, and online survey form, so this provides some useful guide rails for a response.

**All responses must be submitted by February 28, 2023 through the online survey form**

**What is being proposed?**

1. Briefly discuss what is needed to make a submission (oversight commity, steerring commitee, etc)
2. Discuss whether and how to allocate any resources to this, for getting member feedback, and if existing members can contribute

**Comment by seanmcilroy29** [comment]
Chris / Lisa - Review the [Template for submitting proposals related to GHG Protocol’s Corporate Standard, Scope 2 Guidance, Scope 3 Standard, Scope 3 Calculation Guidance and market-based accounting approaches](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fghgprotocol.org%2Fsites%2Fdefault%2Ffiles%2FProposal%2520Template.docx&wdOrigin=BROWSELINK)

**The survey is open until February 28, 2022**

- [ ] Forward Sean questions to used for Survey Monkey
- [ ] Outline who should receive the survey

**Comment by mrchrisadams** [comment]
thanks @seanmcilroy29 we have call booked for next Monday, and I'll update the issue accordingly then 👍

**Comment by mrchrisadams** [comment]
I'm posting this here for folks following along:

> **Dear GSF Standards and Policy WG Members:**
>  
> **TLDR:** 
> The Policy WG are responding to a request from the WRI for input into future versions of the GHG protocol. 
> The GHG protocol is the method 90% of Fortune 500 companies use to calculate their carbon emissions.
> The ask is for you to complete this brief 4 question survey - [Click here ](https://www.research.net/r/sci-ghg-survey)
> The Policy WG will take the responses and create a proposal and submit it to the WRI.
> If you would like to be part of the conversation for how that proposal is formed, please [join the policy-wg](https://groups.google.com/my-groups?hl=en-GB) mailing list and meetings.
> 
> **Background**
> The World Resources Institute (WRI), along with the World Business Council on Sustainable Development (WBCSD), created the GHG Protocol as an international standard for corporate accounting and reporting emissions. In January 2023, [the WRI/WBCSD issued a call for responses](https://ghgprotocol.org/survey-need-ghg-protocol-corporate-standards-and-guidance-updates) regarding potential updates, amendments, or additional guidance to Scope 2 Guidance, by providing the information requested in [this proposal template](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fghgprotocol.org%2Fsites%2Fdefault%2Ffiles%2FProposal%2520Template.docx&wdOrigin=BROWSELINK).
>  
> **How will GSF member survey responses be used in the WRI proposal?** 
> The survey responses provide a pulse check for the Policy WG on how members are using and considering SCI today, along with potential challenges associated with using it for sustainability reporting. If members provide consent. The Policy working group will anonymously aggregate responses to complete proposal questions 1-4, and 8 in the proposal template, located [here](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fghgprotocol.org%2Fsites%2Fdefault%2Ffiles%2FProposal%2520Template.docx&wdOrigin=BROWSELINK). 
>  
> **Survey Timeline**
> GSF internal survey closes on February 22, 2023
> Compilation of GSF survey results week of February 27, 2023
> Internal prereview of aggregated responses on March 6, 2023 (at Policy WG mtg)
> Preparation of WRI proposal using aggregated responses week of March 6, 2023
> Proposal due to WRI on March 14, 2023
> 
> Sincerely,
> 
> Chris Adams and Elise Zelechowski
> Policy Working Group Co-Chairs

**Comment by Henry-WattTime** [comment]
Hi @mrchrisadams are you able to share the final comments you submitted?

**Comment by mrchrisadams** [comment]
hi Henry, sure - sorry about the slow response.

Here's the link to the final submitted response file.

[GSF Response Proposal 2023-03.docx.pdf](https://github.com/Green-Software-Foundation/policy-wg/files/11015054/GSF.Response.Proposal.2023-03.docx.pdf)

I'll check with @elisezelechowski about getting all the content in one place, as I don't think it's all in the same green software foundation G drive right now.

**Comment by seanmcilroy29** [comment]
completed
